src/importer/ImportSession.ts

```diff
@@ -82,7 +82,9 @@ export class ImportSession extends TypedEventManager<IImportSessionEvents> {
       chunkSize: options?.chunkSize || 100,
       chunkTimeout: options?.chunkTimeout || 3000,
     })
-    await chunkIterator.process()
+    await chunkIterator
+      .process()
+      .catch(() => console.warn('Something went wrong. Records processing was stopped.'))
```

Comment:
When data processing is stopped because of a timeout error, RecordChunkIterator emits an error that is bubbled up to the Flatfile instance, so no reason to emit an error again here. However, we still need to catch this error.